data/json/monsters/nether.json

```diff
@@ -1970,5 +1970,137 @@
     "vision_day": 50,
     "vision_night": 8,
     "armor": { "bash": 8 }
+  },
+  {
+    "id": "mon_nether_unviable_butterfly",
+    "type": "MONSTER",
+    "name": { "str": "veiled butterfly", "str_pl": "veiled butterflies" },
+    "description": "There is barely anything there, but you do notice hints of a faint, translucent veiled creature that you can only describe as butterfly-like.  While strangely alluring, it seems unsuited to the atmosphere and is slowly falling apart.",
+    "species": "INSECT",
+    "hp": 20,
+    "speed": 85,
+    "diff": 2,
+    "weight": "200 g",
+    "volume": "5 L",
+    "symbol": "l",
+    "color": "light_red",
+    "morale": 10,
+    "default_faction": "nether",
+    "bodytype": "flying insect",
+    "anger_triggers": [ "HURT" ],
+    "placate_triggers": [ "PLAYER_WEAK" ],
+    "regen_morale": true,
+    "special_attacks": [
+      {
+        "type": "spell",
+        "id": "veiled_butterfly_irradiate",
+        "spell_data": { "id": "veiled_butterfly_irradiate", "min_level": 1 },
+        "cooldown": 30
+      }
+    ],
+    "death_function": { "message": "The %s fades away into nothingness.", "corpse_type": "NO_CORPSE" },
+    "flags": [ "FLIES", "HIT_AND_RUN", "NO_BREATHE", "NO_FUNG_DMG", "SEES" ]
+  },
+  {
+    "id": "mon_nether_unviable_root",
+    "type": "MONSTER",
+    "name": { "str": "writhing root" },
+    "description": "A small chunk of curious looking roots, constantly trying to grow and expand but failing miserably.  It has grown weak, as if incompatible with the very soil it is in.",
+    "species": "PLANT",
+    "hp": 20,
+    "speed": 20,
+    "diff": 2,
+    "weight": "1 kg",
+    "volume": "2 L",
+    "default_faction": "nether",
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "fear_triggers": [ "FIRE" ],
+    "special_attacks": [ { "id": "grab" } ],
```

Comment:
If you want them to be able to grab you you want a non-zero `grab_strength` in the mondef, though if the intention is just to have them flail around that works as well.